service/src/main/java/dk/dbc/rr/oai/OaiBean.java

```diff
@@ -120,8 +120,7 @@ public Response oai(@Context UriInfo uriInfo,
         OaiResponse response = oaiIO.oaiResponseOf(config.getExposedUrl(), params);
         OaiRequest request = response.getRequest();
         try {
-            if (request != null) {
-                log.debug("request.getVerb = {}", request.getVerb());
+            if (request.getVerb() != null) {
```

Comment:
Request has been changed so that it isn't a null any more, due to the case where there are simple parsing generated errors and logic generates errors. In that case all errors are expected to be described in the response.